docs/quickstarts/cosign-akv-azure.md

```diff
@@ -0,0 +1,441 @@
+---
+title: Ensure that AKS clusters only run images signed with Cosign signatures and keys in AKV
+---
+
+# Ensure that AKS clusters only run images signed with Cosign signatures and keys in AKV
+
+This article provides an e2e walkthrough of the process to ensure that Azure Kubernetes Service (AKS) clusters run images that are signed with Cosign signatures and keys stored in Azure Key Vault (AKV).
+
+In this article:
+
+* Sign an image in Azure Container Registry (ACR) with Cosign signatures and keys in AKV
+* Set up an AKS cluster without Azure policy enabled
+* Install OPA Gatekeeper and Ratify
+* Set up Ratify for validating Cosign signatures using keys in AKV
+* Run signed and un-signed images to confirm only signed images are allowed for deployment
+
+> For AKS clusters with Azure policy enabled, Image integrity policy is recommended for ensuring only signed images are deployed
```

Comment:
This is slightly misleading since Image Integrity has notation-only support?

Reply:
True, this is more for image integrity GA according to their plan, I will rephase it.